cfpb/cfgov-refresh
# Fix for footer( Youtube, Flickr) external links routing

Fix for footer( Youtube, Flickr) external links routing.


## Changes

- Modified the Youtube / Filckr links to call the `unsigned_redirect ` method.

## Testing

- Click on the `Youtube` link in the footer; you should be routed to the external redirect page.
- Click on the `Filckr` link in the footer; you should be routed to the external redirect page.

## Checklist
